### ISSUE TITLE
error in box correlation process.

### ISSUE THREAD

**Issue by Mollylulu** [tusen-ai/MV2D#9]
Hi there, thanks for sharing the code. 

I've been trying to run the evaluation using the provided checkpoint, but encountered an error in the **box correlation process**.  Here is the error message:
```bash
MV2D/mmdet3d_plugin/models/roi_heads/utils/box_correlation.py", line 353, in epipolar_in_box
    t_rois_xymax = t_points_xymax.max(1)[0]
RuntimeError: cannot perform reduction function max on tensor with no elements because the operation does not have an identity
```
I was wondering if you've come across this error during your testing, and if so, do you have any suggestions on how to resolve it? 
Thanks in advance.

**Comment by wangzt-halo** [comment]
It seems like the first dimension of `t_points_xymax ` is 0. I think you can add a dimension check before `box_correlation` if you encounter this issue.

**Comment by Mollylulu** [comment]
Indeed, `t_points_xymax` is empty. I encountered this issue while using the provided checkpoint and code for evaluation. Have you come across this problem too, and if so, do you happen to know a solution?

**Comment by wangzt-halo** [comment]
> Indeed, `t_points_xymax` is empty. I encountered this issue while using the provided checkpoint and code for evaluation. Have you come across this problem too, and if so, do you happen to know a solution?

I haven't met this issue so far, but I think you can try to add the code below at Line 304 to see if it works.
If you encounter any problems, please let me know.
```
if not epipolar_in_rois.any():
    corr = rois.new_zeros((num_rois, 0), dtype=torch.int64)
    mask = rois.new_zeros((num_rois, 0), dtype=torch.bool)
    return corr, mask
```
